### PR TITLE
bottom 0.9.5

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
Not the latest release yet: https://github.com/ClementTsang/bottom/releases

**Comment by ClementTsang** [comment]
Sorry about that, just updated it to latest. You might need to update the SHA as well.

**Comment by ZhongRuoyu** [comment]
Thanks for that, @ClementTsang.

File: Formula/b/bottom.rb [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
   head "https://github.com/ClementTsang/bottom.git", branch: "master"
 
+  # There can be a notable gap between when a version is tagged and a
+  # corresponding release is created, so we check the "latest" release instead
+  # of the Git tags.
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```
Comment (ZhongRuoyu) [inline Formula/b/bottom.rb:16]: Added a livecheck.

**Review by p-linnane — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**